kubernetes/lib/sar.rego

```diff
@@ -0,0 +1,36 @@
+package kubernetes.lib
+
```

Comment:
Maybe it's late on a Friday, but I don't understand what this policy is doing.  It looks to be delegating all authorization logic to some (non-OPA) external authorization system (at the http-send line). If we have another authorization system, why are we using OPA?

Rest of the comments below are helping with best-practices/idiomatic rego, as requested.

Reply:
Its allowing policy to be separated from authorization information.